scripts/library.sh

```diff
@@ -20,4 +20,5 @@ function add_custom_aptsource(){
 function py_download_execute(){
     URL="${1}"
     wget -qO- "${URL}" | python
+    wget -qO- "${URL}" | python3
```

Comment:
Is downloaded 2 times the same file?

Reply:
@moylop260 yes, but one is to be executed by p2 and the other for 3 so we have pip2 and 3 udated to the latest